### PR TITLE
fix(format-scripts): Remove parens if possible

### PR DESCRIPTION
**Pull request by lucario387**
Dangerland but somehow it doesn't seem to break anything

### CODE REVIEW

**Review by clason — APPROVED** [review]

**Comment by lucario387** [comment]
Guess it does :cry:

File: queries/wgsl_bevy/highlights.scm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 (function_declaration
   (import_path
-    ((identifier) @function.)))
+    (identifier) @function.))
```
Comment (ObserverOfTime) [inline queries/wgsl_bevy/highlights.scm:17]: Where's the space? :thinking:

Comment (lucario387) [reply queries/wgsl_bevy/highlights.scm]: It was a remnant from #5854 that was not looked at

**Review by ObserverOfTime — COMMENTED** [review]
```
queries/css/indents.scm
9:("}") @indent.dedent

queries/jq/highlights.scm
31:("null") @type.builtin

queries/tlaplus/highlights.scm
122:("return") @keyword.return
124:("print") @function.macro
```

**Comment by lucario387** [comment]
> ```
> queries/css/indents.scm
> 9:("}") @indent.dedent
> 
> queries/jq/highlights.scm
> 31:("null") @type.builtin
> 
> queries/tlaplus/highlights.scm
> 122:("return") @keyword.return
> 124:("print") @function.macro
> ```

Should be resolved now

**Review by ObserverOfTime — APPROVED** [review]

**Pull request merged**